brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
http://babeljs.io/blog/2015/10/29/6.0.0/

Pretty big chunk of "benefits." I guess the question for me is, Do those benefits out weight IE 8 support? For me, the answer would be "yes." But I'm also not concerned with Internet Explorer.